src/status_im/chat/views/toolbar_content.cljs

```diff
@@ -52,8 +52,7 @@
     [last-activity {:sync-state sync-state}]
     (if public?
       [view {:flex-direction :row}
-       [text {:style (get-in platform-specific [:component-styles :toolbar-last-activity])}
-        (label :t/public-group-status)]]
+       [text (label :t/public-group-status)]]
```

Comment:
No more style?

Reply:
It was not defined in old platform-specific styles, so I think it can be safely removed here.

Reply:
👍